lib/import/rummager_data_presenter.rb

```diff
@@ -7,6 +7,7 @@ def self.present_content(batch_data)
           item['content_id'],
           item['format'],
           item['index'],
+          item['is_withdrawn'] == 'true' ? 'withdrawn' : 'not_withdrawn',
```

Comment:
Not at all, if you saw that in the console then it should be right - I'm just surprised it ended up like that as we treat it as a boolean in rummager:

```ruby
withdrawn_query = { "not" => { "term" => { "is_withdrawn" => true } } }
```

Can't comment much on ruby idioms, but module of constants is top answer on stack overflow so sounds reasonable! http://stackoverflow.com/questions/75759/enums-in-ruby